### PR TITLE
Serialization_test quick fix (remove exapi)

### PR DESCRIPTION
**Pull request by mwielgus**
Temporary fix for #13083

### CODE REVIEW

File: pkg/api/serialization_test.go [hunk 1]
```diff
@@ -32,8 +32,9 @@ import (
 	"k8s.io/kubernetes/pkg/runtime"
 	"k8s.io/kubernetes/pkg/util"
 
-	_ "k8s.io/kubernetes/pkg/expapi"
-	_ "k8s.io/kubernetes/pkg/expapi/v1"
+	// TODO: enable when exapi problems are fixed
```
Comment (wojtek-t) [inline pkg/api/serialization_test.go]: Can you please create an issue for it and add a link here?

Comment (mwielgus) [inline pkg/api/serialization_test.go]: Added to #13083.

**Comment by wojtek-t** [comment]
LGTM

**Comment by mwielgus** [comment]
Jenkins E2E failure is related to #13032, merging.

**Pull request merged**